ably/realtime_client.go

```diff
@@ -94,6 +94,8 @@ func (c *RealtimeClient) onReconnectMsg(msg *proto.ProtocolMessage) {
 		}
 		if msg.Error != nil {
 			// (RTN15c3)
+			// process all queued messages. No state changes here.
```

Comment:
I interpret [RTN15c3](RTN15c3) as also moving to CONNECTED; the item doesn't really say otherwise, and it's arguably implied by the "CONNECTED event" it talks about. Since it doesn't specify any other transition, and also because it makes sense (we're definitely CONNECTED, although on a different connection), the default transition to CONNECTED should apply here.

Plus, this is missing a lot from RTN15c3. Just commenting this to avoid losing track of that, since there's no placeholder comment for all that.